src/hw9/ArrayDataException.java

```diff
@@ -1,6 +1,6 @@
 package hw9;
 
-public class ArrayDataException extends Throwable{
-    public ArrayDataException ( String s ){
+public class ArrayDataException extends Throwable {
+    public ArrayDataException (String s){
```

Comment:
If you're overriding a constructor and it accepts a parameter, then the constructor's body should have this line `super(s)`, please add it.